3party/osrm/osrm-backend/plugins/MapsMePlugin.hpp

```diff
@@ -29,6 +29,64 @@
 #include <string>
 #include <vector>
 
+double constexpr kMaxDistanceToFindMeters = 1000.0;
```

Comment:
This constant is used only once, how about to move it to the method where it's used?